server/src/main/java/org/spine3/server/entity/FieldMasks.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2016, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.entity;
+
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.FieldMask;
+import com.google.protobuf.Message;
+import com.google.protobuf.ProtocolStringList;
+import org.spine3.protobuf.KnownTypes;
+import org.spine3.protobuf.TypeUrl;
+
+import javax.annotation.Nullable;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * <p>Util class for filtering fields that are included in an instance of {@link Message} or collection of {@link Message}s.
+ * Provides basic functionality for processing of e.g. a {@code org.spine3.client.Query} to in-memory storage.</p>
+ *
+ * @author Dmytro Dashenkov
+ */
+@SuppressWarnings("UtilityClass")
+public class FieldMasks {
+
+    private FieldMasks() {
+    }
+
+    /**
+     * <p>Applies given {@code FieldMask} to givem collection of {@link Message}s.
+     * Does not change the {@link Collection} itself.</p>
+     *
+     * <p>The {@code FieldMask} must be valid for this operation.</p>
```

Comment:
Here and through the code: we follow HTML 1.1 and we do not close `<p>`. 

Please check [https://github.com/SpineEventEngine/core-java/wiki](url) if you haven't already.